### PR TITLE
Docs for newly added option (`wipeMode`) to quarantine procedure

### PR DESCRIPTION
**Pull request by Frannie-Ludmilla**
New signature for the quarantine procedure. Relates to https://github.com/neo-technology/neo4j/pull/27340

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1881.surge.sh

Updated pages:
- [Procedures](https://neo4j-docs-operations-1881.surge.sh/operations-manual/5/reference/procedures)

File: modules/ROOT/pages/reference/procedures.adoc [hunk 1]
```diff
@@ -2044,13 +2044,21 @@ For more information, see the link:{neo4j-docs-base-uri}/cypher-manual/{page-ver
...
 m|DBMS
 // | Default roles
 // m|admin
 |===
 
+[NOTE]
+====
+The `wipeMode` option is only enforced during the removal of a database from quarantine.
+The possible values for `wipeMode` are: `DO_NOT_WIPE` (default value and previous behaviour), `WIPE_STATE_KEEP_STORE` and `WIPE_STATE_AND_STORE`.
+Please keep in mind that the deletion of the cluster state and store (`WIPE_STATE_AND_STORE`) or even just the deletion of the state (`WIPE_STATE_KEEP_STORE`) could cause loss of data and should be carefully evaluated depending on the case.
```
Comment (tselmegbaasan) [inline modules/ROOT/pages/reference/procedures.adoc:2058]: Maybe it's worth explaining that wiping the cluster state means this server will join the cluster as a new member. And in order for the cluster to accept new members, they need to have a quorum of old members. And therefore it's dangerous to wipe the state for >=quorum of members. Sometimes the quarantine might be due to the store state, so deleting the store only might be sufficient. But I see now that we don't have an option for it, let's discuss it.

**Comment by NataliaIvakina** [comment]
Hi! @Frannie-Ludmilla, could you please rebase your PR with the latest version of the `dev` branch? You PR misses so many procedures and current changes.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1881.surge.sh

Updated pages:
- [Procedures](https://neo4j-docs-operations-1881.surge.sh/operations-manual/5/reference/procedures)

**Pull request closed without merging**